test/integration/options.spec.js

```
@@ -346,6 +346,26 @@ describe('options', function() {
           done();
         });
       });
+
+      it('fails if no --grep', function(done) {
```

`should throw an error ...`.

---

```js
      it('should throw an error when `--invert` used in isolation', function(done) {
```